refinery/core/api.py

```diff
@@ -699,7 +699,7 @@ def get_by_db_id(self, request, **kwargs):
                 is_public = True
 
         # get_owner in core models uses add to distinguish owner
```

Comment:
add_%s perms is used for checking ownership on non-shareable resources like analyses and workflows
data_set uses share_%s (only owners can share data_sets right now)